app/views/users/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1>Create new User</h1>
+<% if false %> 
+<form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username</label>
+  <input type="text" name="user[username]" id="username"><br><br>
+
+  <label for="email">Email</label>
+  <input type="email" name="user[email]" id="email"><br><br>
+
+  <label for="password">Password</label>
+  <input type="password" name="user[password]" id="password"><br><br>
+
+  <input type="submit" value="Submit">
+</form>
+<% end %> 
+
+<% if false %> 
+<%= form_tag("/users") do %>
+  <%= label_tag(:username, "Username")%>
+  <%= text_field_tag(:username)%>
+    <br>
```

Comment:
- [ ] This piece [should](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) be commented out